icdc_schema.py

```diff
@@ -265,7 +266,7 @@ def get_type(self, name):
                         if ITEM_TYPE in prop_desc:
                             item_type = self._get_item_type(prop_desc[ITEM_TYPE])
                             result[ITEM_TYPE] = item_type
-                        elif PROP_ENUM in prop_desc:
+                        if PROP_ENUM in prop_desc:
```

Comment:
I wonder, what's the reason for this change?

Reply:
There might be possible that some properties have both ITEM_TYPE and PROP_ENUM